yt_dlp/extractor/neteasemusic.py

```diff
@@ -203,91 +154,151 @@ class NetEaseMusicIE(NetEaseMusicBaseIE):
-            'alt_title': alt_title,
-            'creator': ' / '.join([artist['name'] for artist in info.get('artists', [])]),
-            'timestamp': self.convert_milliseconds(info.get('album', {}).get('publishTime')),
-            'thumbnail': info.get('album', {}).get('picUrl'),
-            'duration': self.convert_milliseconds(info.get('duration', 0)),
-            'description': lyrics,
             'formats': formats,
+            'alt_title': '/'.join(traverse_obj(info, (('transNames', 'alias'), ...))) or None,
+            'creator': ' / '.join(traverse_obj(info, ('artists', ..., 'name'))),
+            **lyric_data,
+            **traverse_obj(info, {
+                'title': ('name', {str}),
+                'timestamp': ('album', 'publishTime', {lambda i: int_or_none(i, scale=1000)}),
+                'thumbnail': ('album', 'picUrl', {url_or_none}),
+                'duration': ('duration', {lambda i: int_or_none(i, scale=1000)}),
+            }),
         }
 
 
 class NetEaseMusicAlbumIE(NetEaseMusicBaseIE):
     IE_NAME = 'netease:album'
     IE_DESC = '网易云音乐 - 专辑'
     _VALID_URL = r'https?://music\.163\.com/(#/)?album\?id=(?P<id>[0-9]+)'
-    _TEST = {
+    _TESTS = [{
+        'url': 'https://music.163.com/#/album?id=133153666',
+        'info_dict': {
+            'id': '133153666',
+            'title': '桃几的翻唱',
+            'upload_date': '20210913',
+            'description': '桃几2021年翻唱合集',
+            'thumbnail': r're:^http.*\.jpg',
+        },
+        'playlist_mincount': 13,
+    }, {
         'url': 'http://music.163.com/#/album?id=220780',
         'info_dict': {
             'id': '220780',
-            'title': 'B\'day',
+            'title': 'B\'Day',
+            'upload_date': '20060904',
+            'description': 'md5:71a74e1d8f392d88cf1bbe48879ad0b0',
+            'thumbnail': r're:^http.*\.jpg',
         },
         'playlist_count': 23,
-        'skip': 'Blocked outside Mainland China',
-    }
+    }]
 
     def _real_extract(self, url):
         album_id = self._match_id(url)
-
-        info = self.query_api(
-            'album/%s?id=%s' % (album_id, album_id),
-            album_id, 'Downloading album data')['album']
-
-        name = info['name']
-        desc = info.get('description')
+        webpage = self._download_webpage(f'https://music.163.com/album?id={album_id}', album_id)
+
+        songs = self._search_json(
+            r'<textarea id="song-list-pre-data" style="display:none;">', webpage, 'metainfo', album_id,
+            end_pattern=r'</textarea>', contains_pattern=r'\[(?s:.+)\]')
+        metainfo = {
+            'title': self._og_search_property('title', webpage, 'title', fatal=False),
+            'description': clean_html(self._search_regex(
+                r'(?:<div id="album-desc-dot".*)?<div id="album-desc-(?:dot|more)"[^>]*>(.*?)</div>', webpage,
+                'description', flags=re.S, fatal=False)),  # match album-desc-more or fallback to album-desc-dot
```

Comment:
`_html_search_regex` does the `clean_html` for you

also, maybe 2 separate patterns would be cleaner? though I don't know what the source looks like

Reply:
For shorter desc, the source looks like
`<div id="album-desc-dot">full description</div> {other contents}`
For longer desc, the source looks like
`<div id="album-desc-dot">descripti...</div><div id="album-desc-more">description in full</div> {other contents}`
Splitting into two patterns will be cleaner, but I was concern that the first pattern would raise a warning if it fails to find `album-desc-more`, which is expected for shorter descriptions.

Reply:
`_html/_search_regex`'s first argument can be a tuple of regex patterns; if done that way it will not warn when falling back to the second pattern

Reply:
Most of two patterns are the same, so I tried using a f-string for patterns: https://github.com/yt-dlp/yt-dlp/pull/8181/commits/5efc61346bfd316b72b136f848db474c6239f6b7
Does this looks good to you?

Reply:
https://github.com/yt-dlp/yt-dlp/pull/8181/files#r1348083829